3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,136 @@
+//--------------------------------------------------exercise 1
+//Remove the item in names that is equal to nameToRemove
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  let index = names.indexOf("Ahmad");
+  console.log(index); //index "Ahmad" in the array 1
+  if (index > 0){
```

Comment:
i would check here like this : `if(index>=0)`, because the first element in the array has 0 index, and maybe you would want to remove it in future 😸